.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "endOfLine": "lf",
+  "semi": false,
+  "singleQuote": true,
+  "trailingComma": "es5"
+}
```

Comment:
most of our code uses 4 space for tab width. I think we should keep it.

Reply:
I will update this settings, by default it is 2.